src/datastore/common.ts

```diff
@@ -38,6 +38,7 @@ export enum DbTxStatus {
   Success = 1,
   AbortByResponse = -1,
   AbortByPostCondition = -2,
+  Empty = -3,
```

Comment:
Same question here -- what is the purpose of this flag?

Reply:
@fdmirza